### PR TITLE
Show gallery in updates feed

### PR DESCRIPTION
**Pull request by burukin-wizards**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/components/Story/StoryPreview.js [hunk 1]
```diff
@@ -23,8 +23,15 @@ const StoryPreview = ({ post }) => {
...
   let imagePath = '';
 
-  if (jsonMetadata && jsonMetadata.image && jsonMetadata.image[0]) {
-    imagePath = getProxyImageURL(jsonMetadata.image[0], 'preview');
+  if (jsonMetadata) {
+    if (jsonMetadata.image && jsonMetadata.image[0]) {
+      imagePath = getProxyImageURL(jsonMetadata.image[0], 'preview');
+    } else if (
+      jsonMetadata.wobj &&
+      _.includes(['galleryItem', 'avatar', 'background'], jsonMetadata.wobj.field.name)
```
Comment (milklim) [inline src/client/components/Story/StoryPreview.js]: check if 'field' property exist

File: src/client/object/wObjectHelper.js [hunk 2]
```diff
@@ -221,9 +221,12 @@ export const hasActionType = (post, actionTypes = ['createObject', 'appendObject
   );
 };
 
-export const mapObjectAppends = (comments, wObj) => {
+export const mapObjectAppends = (comments, wObj, albums) => {
+  const galleryImages = [];
+  albums.map(album => album.items.map(item => galleryImages.push(item)));
```
Comment (milklim) [inline src/client/object/wObjectHelper.js]: replace .map() with .forEach()

File: src/client/components/Story/StoryPreview.js (hunk 1 above)
Comment (burukin-wizards) [reply src/client/components/Story/StoryPreview.js]: Done

File: src/client/object/wObjectHelper.js (hunk 2 above)
Comment (burukin-wizards) [reply src/client/object/wObjectHelper.js]: Done

File: src/client/object/ObjectGallery/CreateAlbum.js [hunk 3]
```diff
@@ -1,11 +1,60 @@
...
+const CreateAlbum = ({
+  showModal,
+  hideModal,
+  form,
+  loading,
+  intl,
+  currentUsername,
+  wObject,
+  // eslint-disable-next-line no-shadow
+  appendObject,
```
Comment (milklim) [inline src/client/object/ObjectGallery/CreateAlbum.js]: better rename 'appendObject' props to 'dispatchAppendObject' or 'appendObjectDispatch' and remover eslint comment

File: src/client/object/ObjectGallery/CreateAlbum.js [hunk 4]
```diff
@@ -1,11 +1,60 @@
...
+  // eslint-disable-next-line no-shadow
+  addAlbumToStore,
```
Comment (milklim) [inline src/client/object/ObjectGallery/CreateAlbum.js]: the same

File: src/client/object/ObjectGallery/CreateAlbum.js (hunk 3 above)
Comment (burukin-wizards) [reply src/client/object/ObjectGallery/CreateAlbum.js]: Done

File: src/client/object/ObjectGallery/CreateAlbum.js (hunk 4 above)
Comment (burukin-wizards) [reply src/client/object/ObjectGallery/CreateAlbum.js]: Done

File: src/client/object/WobjHistory.js [hunk 5]
```diff
@@ -105,14 +118,27 @@ export default class WobjHistory extends React.Component {
   };
 
   handleToggleModal = () => {
-    this.setState({ showModal: !this.state.showModal });
+    if (this.state.field === 'galleryItem') {
```
Comment (milklim) [inline src/client/object/WobjHistory.js]: use constant - objectFields.galleryItem

File: src/client/object/WobjHistory.js [hunk 6]
```diff
@@ -105,14 +118,27 @@ export default class WobjHistory extends React.Component {
...
+      this.setState(prevState => ({ showModalGalleryItem: !prevState.showModalGalleryItem }));
+    } else if (this.state.field === 'galleryAlbum') {
```
Comment (milklim) [inline src/client/object/WobjHistory.js]: use constant - objectFields.galleryAlbum

Comment (burukin-wizards) [reply src/client/object/WobjHistory.js]: Done

**Pull request merged**